modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractKotlinCodegen.java

```diff
@@ -176,6 +176,7 @@ public AbstractKotlinCodegen() {
         typeMapping.put("binary", "kotlin.Array<kotlin.Byte>");
         typeMapping.put("Date", "java.time.LocalDate");
         typeMapping.put("DateTime", "java.time.LocalDateTime");
+        typeMapping.put("ref", "kotlin.Any");
```

Comment:
@gimlet2 I don't think there's a type called "ref".

Reply:
Hi @wing328! Yes, there is no such type. But in spec I've got i see `{
                        "name": "request",
                        "in": "formData",
                        "description": "request",
                        "required": false,
                        "type": "ref"
                    }`
And that was generated by `springfox` swagger. I think that can be fixed some other way and will be happy to do it if you can recommend me how.

Reply:
I think it's bug in springfox based on the information you've provided as `type: ref` is not defined in the official openapi/swagger spec.

Given that it's a form parameter, what exactly is that? string? float?

What does the HTTP request payload look like?

Reply:
It looks like `ref` is default type https://github.com/swagger-api/swagger-core/blob/v1.5.21/modules/swagger-models/src/main/java/io/swagger/models/properties/RefProperty.java But it was changed in lates versions. Maybe that will help

Reply:
`ref` is not defined in the OpenAPI/Swagger spec. Please use a validator to validate the API spec/document first.

Reply:
I don't think we should include this in the generator, and rather require those who need it to pass this as a type mapping at generation time (see CLI options for example.

A `ref` in OpenAPI means something completely different than a top-level object type. Adding this means we need to support it through a deprecation cycle, and it may cause problems for others.

Reply:
Agree. Removed